lib/beaker/options/presets.rb

```diff
@@ -185,6 +185,7 @@ def presets
           :puppetdb_port_nonssl   => 8080,
           :puppetserver_port      => 8140,
           :nodeclassifier_port    => 4433,
+          :cache_files_locally    => true, # TODO change to false in next major version
```

Comment:
If we're adding values to `presets`, we should probably bring in this spec which tests our defaults: https://github.com/puppetlabs/beaker/pull/1013/files#diff-e00c6e55b14629bb263ac9d4b5fb00b2

It was reverted along with the reversion of the `TestCase` refactoring for alternate test runners; it could be reintroduced here, or added in a separate PR.